packages/geo/src/lib/datasource/shared/datasources/datasource.interface.ts

```diff
@@ -25,6 +25,8 @@ export interface DataSourceOptions {
   // TODO: Should those options really belong here?
   sourceFields?: SourceFieldsOptionsParams[];
   download?: DownloadOptions;
+  url?: string;
```

Comment:
Should be put only on the types of datasources supporting an offline url.

Reply:
the sourceOptions layer from the map is attributed to dataSourceOptions and not from the other datasource type. I can't can get url or path from a datasource if url is not in dataSourceOptions